src/PHPSemVerChecker/Operation/ClassRenamedCaseOnly.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace PHPSemVerChecker\Operation;
+
+use PhpParser\Node\Stmt\Class_;
+use PHPSemVerChecker\Node\Statement\Class_ as PClass;
+
+class ClassRenamedCaseOnly extends Operation {
```

Comment:
@tomzx should this extend from a new class called `ClassOperationDelta` or `ClassOperationUnary` or something similar?

Right now the constructor for `ClassRenamedCaseOnly` is `public function __construct($fileAfter, Class_ $classAfter)`, is this correct?

Reply:
Yes, all of the operations should extend either the unary of delta variant. But this means it would require the creation of these classes for `class`, `trait`, `interface` as well. I guess this can be done later on in a refactoring pass.

Reply:
I've just pushed @ 7944fd1c04f93eeeacbfe483c26de715c75a741f which creates said classes.